libs/core/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -121,41 +127,47 @@ export class BreadcrumbComponent implements AfterViewInit, OnInit, OnDestroy {
         this._subscriptions.unsubscribe();
     }
 
-    /** @hidden */
+    /**
+     * Callback function when breadcrumbs container has been resized.
+     */
     onResize(): void {
         if (!this.elementRef.nativeElement.parentElement) {
             return;
         }
-        this.containerBoundary = this.elementRef.nativeElement.parentElement.getBoundingClientRect().width;
+        this._containerBoundary = this.elementRef.nativeElement.parentElement.getBoundingClientRect().width;
 
         if (this.containerElement) {
-            this.containerBoundary = this.containerElement.getBoundingClientRect().width;
+            this._containerBoundary = this.containerElement.getBoundingClientRect().width;
+        }
+
+        if (this._overflowContainer) {
+            this._containerBoundary -= this._overflowContainer.nativeElement.getBoundingClientRect().width;
         }
 
         let visibleSum = 0;
-        const breadcrumbItemComponents = this.breadcrumbItems.toArray();
+        const breadcrumbItemComponents = this._breadcrumbItems.toArray();
         let i;
         for (i = breadcrumbItemComponents.length - 1; i >= 0; i--) {
```

Comment:
it looks like overflow layout algorithm, isn't it? could we use it here instead of having this code?